templates/.goreleaser.yml.tpl

```
@@ -29,7 +29,12 @@ builds:
    {{- end }}
   env:
   - CGO_ENABLED={{ stencil.ApplyTemplate "cgoEnabled" | trim }}
-{{- end }}
+  {{- $blockName := (printf "%vAdditionalEnv" $cmdName) }}
+  ## <<Stencil::Block({{ $blockName }})>>
+  {{- $env := (file.Block $blockName ) | trim }}
+{{ $env | indent 2}}
```

suggestion: does it still work if you merge this into one line?
```suggestion
{{ (file.Block $blockName) | trim | indent 2 }}
```

---

> (file.Block $blockName)

Good catch. $env was just initially created to check if the block is empty. Applied the change.